### PR TITLE
Replace bullet point with numbered list for notes

### LINKED ISSUE #194: Update notes in contact details to be a numbered list

### PR DESCRIPTION
**Pull request by ITLimJiaWei**
image

Closes #194

### CODE REVIEW

**Review by sheenkerr — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/ui/ContactDetails.java [hunk 1]
```diff
@@ -91,12 +90,14 @@ private void setPanelInformation() {
                 Label notesHeader = new Label("Notes");
                 notesHeader.setId("notes-header");
                 notesList.getChildren().add(notesHeader);
+
+                int[] index = {1};
```
Comment (KennethTeo2002) [inline src/main/java/seedu/address/ui/ContactDetails.java:94]: Consider using `AtomicInteger`

File: src/main/java/seedu/address/ui/ContactDetails.java [hunk 2]
```diff
@@ -91,12 +90,14 @@ private void setPanelInformation() {
...
                 person.getNotes().stream()
-                        .sorted(Comparator.comparing(note -> note.getNote()))
                         .forEach(note -> {
-                            Label label = new Label("• " + note.getNote());
+                            Label label = new Label(index[0] + ". " + note.getNote());
```
Comment (KennethTeo2002) [inline src/main/java/seedu/address/ui/ContactDetails.java:97]: `AtomicInteger` can use index.getAndIncrement(), so no longer need to increment at the end.

**Review by KennethTeo2002 — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/ui/ContactDetails.java (hunk 2 above)
Comment (ITLimJiaWei) [reply src/main/java/seedu/address/ui/ContactDetails.java:97]: alright, I will update it

**Review by KennethTeo2002 — APPROVED** [review]
LGTM

**Pull request merged**